.env.yaml.template

```diff
@@ -1,3 +1,2 @@
 env_variables:
-  REACT_APP_S3_URL: 
-
+  REACT_APP_S3_URL:
```

Comment:
It'd be cool if you can revert all changes done to this file in such a way that it doesn't show up in the PR